index.html

```diff
@@ -23,28 +23,190 @@
+              <ul class="card-btns">
+                <li class="card-btn">Ruby on Rails</li>
+                <li class="card-btn">CSS</li>
+                <li class="card-btn">Javascript</li>
+                <li class="card-btn">HTML</li>
+              </ul>
+              <div class="cta-btn"><a href="#">See project</a></d>
+            </div>
+          </article>
+          <article class="work-card">
+            <div class="work-card__head"></div>
+            <div class="work-card__body">
+              <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+              <ul class="card-btns">
+                <li class="card-btn">Ruby on Rails</li>
+                <li class="card-btn">CSS</li>
+                <li class="card-btn">Javascript</li>
+                <li class="card-btn">HTML</li>
+              </ul>
+              <div class="cta-btn"><a href="#">See project</a></d>
+            </div>
+          </article>
+          <article class="work-card">
+            <div class="work-card__head"></div>
+            <div class="work-card__body">
+              <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+              <ul class="card-btns">
+                <li class="card-btn">Ruby on Rails</li>
+                <li class="card-btn">CSS</li>
+                <li class="card-btn">Javascript</li>
+                <li class="card-btn">HTML</li>
+              </ul>
+              <div class="cta-btn"><a href="#">See project</a></d>
+            </div>
+          </article>
+          <article class="work-card">
+            <div class="work-card__head"></div>
+            <div class="work-card__body">
+              <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+              <ul class="card-btns">
+                <li class="card-btn">Ruby on Rails</li>
+                <li class="card-btn">CSS</li>
+                <li class="card-btn">Javascript</li>
+                <li class="card-btn">HTML</li>
+              </ul>
+              <div class="cta-btn"><a href="#">See project</a></d>
+            </div>
+          </article>
+          <article class="work-card">
+            <div class="work-card__head"></div>
+            <div class="work-card__body">
+              <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+              <ul class="card-btns">
+                <li class="card-btn">Ruby on Rails</li>
+                <li class="card-btn">CSS</li>
+                <li class="card-btn">Javascript</li>
+                <li class="card-btn">HTML</li>
+              </ul>
+              <div class="cta-btn"><a href="#">See project</a></d>
+            </div>
+          </article>
+        </div>
+      </section>
+  
+      <!-- About me -->
+      <section>
+        <section class="about-me-header">
+          <div class="about-me-header-container">
+            <h2 class="section-title about-me-title">About me</h2>
+            <hp class="about-me-sub-title"> 
```

Comment:
- [ ] I see you tried to make the changes requested to use the `<p>` tag for the paragraph, but I see you were using `<hp>` instead. Kindly change `<hp>` to `<p>` tag